tinyagentos/knowledge_fetchers/youtube.py

```
@@ -14,6 +14,19 @@
 
+# Tracked subprocesses so they can be killed on timeout or cancel.
+_tracked_procs: list[asyncio.subprocess.Process] = []
+
+
+def _cleanup_procs() -> None:
+    """Kill all tracked subprocesses (called on timeout or cancel)."""
+    for p in _tracked_procs:
+        try:
+            p.kill()
+        except Exception:
+            pass
```

Ruff (0.16.0)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)